docs/source/guide/install_enterprise_k8s.md

```diff
@@ -0,0 +1,418 @@
+  extraLabels: { }
+
+  affinity: { }
+
+  # Toleration Settings for rqworker pods
+  # Provide either a multi-line string or YAML matching the Toleration array
+  # in a PodSpec.
+  tolerations: []
+
+  # Used to define custom readinessProbe settings
+  readinessProbe:
+    enabled: false
+    path: /health
+    # When a probe fails, Kubernetes will try failureThreshold times before giving up
+    failureThreshold: 2
+    # Number of seconds after the container has started before probe initiates
+    initialDelaySeconds: 5
+    # How often (in seconds) to perform the probe
+    periodSeconds: 5
+    # Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+    # Number of seconds after which the probe times out.
+    timeoutSeconds: 3
+  # Used to enable a livenessProbe for the pods
+  livenessProbe:
+    enabled: false
+    path: "/health"
+    # When a probe fails, Kubernetes will try failureThreshold times before giving up
+    failureThreshold: 2
+    # Number of seconds after the container has started before probe initiates
+    initialDelaySeconds: 60
+    # How often (in seconds) to perform the probe
+    periodSeconds: 5
+    # Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+    # Number of seconds after which the probe times out.
+    timeoutSeconds: 3
+
+  # Definition of the serviceAccount used to run rqworker for Label Studio Enterprise
+  serviceAccount:
+    # Specifies whether to create a service account
+    create: true
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+    # Extra k8s annotations for the serviceAccount definition. This can either be
+    # YAML or a YAML-formatted multi-line templated string map of the
+    # annotations to apply to the serviceAccount.
+    annotations: { }
+    
+{% endcodeblock %}
+    
+{% enddetails %}
+<br/>
+
+## Install Label Studio Enterprise using Helm on a Kubernetes cluster
+
+Use Helm to install Label Studio Enterprise on your Kubernetes cluster. Provide your custom reource definitions YAML file. Specify any environment variables that you need to set for your Label Studio Enterprise installation using the `--set` argument with the `helm install` command.
+
+From the command line, run the following:
+```shell
+helm install lse . -f lse-values.yaml
+```
+
+After installing, check the status of the Kubernetes pod creation:
+```shell
+kubectl get pods
+```
+
+## Upgrade Label Studio using Helm
+To upgrade Label Studio Enterprise using Helm, do the following.
+
+Run `helm upgrade` with your values YAML file provided:
+```shell
+helm upgrade lse . -f lse-values.yaml
+```
+
+You can also upgrade to the latest version by specifying the latest version in an environment variable:
+```shell
+helm upgrade lse . -f lse-values.yaml --set global.image.tag=<new_version>
```

Comment:
what's the motivation behind this second syntax option?

Reply:
Just quick override, in this case the change of image.tag will not be stored in the lse-values.yaml. btw, I'd like to ommit this line, because we have moved out of the model when we published the "latest" image.

Reply:
perfect, that makes sense to me! I'll remove this option